servo-atlas/src/main/java/com/netflix/servo/publish/atlas/AtlasMetricObserver.java

```diff
@@ -112,11 +116,22 @@ public AtlasMetricObserver(ServoAtlasConfig config,
     this.sendTimeoutMs = stepMs * 9 / 10;
     this.commonTags = commonTags;
     pushQueue = new LinkedBlockingQueue<>(config.getPushQueueSize());
-    final Thread pushThread = new Thread(new PushProcessor(), "BaseAtlasMetricObserver-Push");
+    pushThread = new Thread(new PushProcessor(), "BaseAtlasMetricObserver-Push");
     pushThread.setDaemon(true);
     pushThread.start();
   }
 
+  /**
+   * Stop attempting to send metrics to atlas.
+   * Cleans up the thread that is created by this observer.
+   */
+  public void stop() {
```

Comment:
It's probably a good idea to add it to the base interface.  I'll create a new PR with that change.